x-pack/plugins/canvas/server/usage/__tests__/workpad_collector.handle_response.ts

```diff
@@ -5,12 +5,13 @@
  */
 
 import expect from '@kbn/expect';
-import { handleResponse } from '../collector';
+import { handleResponse } from '../workpad_collector';
+// @ts-ignore Missing local definition
 import { workpads } from '../../../__tests__/fixtures/workpads';
 
-const getMockResponse = (mocks = workpads) => ({
+const getMockResponse = (mocks = workpads): any => ({
```

Comment:
You should put in what you know... in this case, that you're returning `hits`, etc.  The fact the workpad is not typed is fine, but when that is finally typed, that should simply flow through without someone needing to type the other parts.